qiskit_algorithms/time_evolvers/trotterization/trotter_qrte.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         product_formula: ProductFormula | None = None,
         estimator: BaseEstimator | None = None,
         num_timesteps: int = 1,
+        insert_barriers: bool = False,
```

Comment:
```suggestion
        *,
        insert_barriers: bool = False,
```
My tendency would be to have this only as a named argument and not add another positional argument for future flexibility (ie add the * to denote this and require one to use `insert_barriers=True` and not just tag `True` on the end).

Its kinda odd that Args docstring has  num_timesteps listed before estimator whereas in the parameters they are in the opposite order - but it was like that before it seems!

Reply:
I swapped the order of the arguments in [3d3a7c9](https://github.com/qiskit-community/qiskit-algorithms/pull/128/commits/3d3a7c939b35139c7e6a2ba85391f3f607f86eca).